re/re_common/proto/aggregationmessage/aggregationmessage.proto

```diff
@@ -197,6 +206,21 @@ message NetworkUtilisationEvent {
     google.protobuf.Timestamp time = 5;
 }
 
+message NodeGPUMetrics {
+    Node node_info = 1;
+    repeated GPUDeviceMetrics per_device_metrics = 2;
+}
+message GPUDeviceMetrics {
+    int32 gpu_device_index = 1;
+    repeated GPUMetricSample samples = 2;
+}
+message GPUMetricSample {
+    google.protobuf.Timestamp time = 1;
+    double gpu_utilisation = 2;
+    int64 mem_utilisation = 3;
```

Comment:
Was this meant to be a double like the MemoryUtilisationEvent?

Reply:
I'll be receiving the memory usage as the raw number of MiB, so if I rename it to include the units this should hopefully be a little clearer.